README.md

```diff
@@ -49,5 +57,30 @@ Objective functions are created with package [smoof](https://github.com/jakobbos
 Parameter spaces and initial designs are created with package [ParamHelpers](https://github.com/berndbischl/ParamHelpers).
 
 
+# mlrMBO - How to Cite and Citing Publications
+
+Please cite our [arxiv paper](https://arxiv.org/abs/1703.03373) (Preprint).
+You can get citation info via `citation("mlrMBO")` or copy the following BibTex entry:
+
+```bibtex
+@article{mlrMBO,
+  title = {{{mlrMBO}}: {{A Modular Framework}} for {{Model}}-{{Based Optimization}} of {{Expensive Black}}-{{Box Functions}}},
+  url = {http://arxiv.org/abs/1703.03373},
+  shorttitle = {{{mlrMBO}}},
+  archivePrefix = {arXiv},
+  eprinttype = {arxiv},
+  eprint = {1703.03373},
+  primaryClass = {stat},
+  author = {Bischl, Bernd and Richter, Jakob and Bossek, Jakob and Horn, Daniel and Thomas, Janek and Lang, Michel},
+  date = {2017-03-09},
+}
+```
+
+Some parts of the package were created as part of other publications.
+If you use these parts, please cite the relevant work appropriately:
+
+* Multi-point proposals using multi-objective infill criteria: [MOI-MBO: Multiobjective Infill for Parallel Model-Based Optimization](https://dx.doi.org/10.1007/978-3-319-09584-4_17)
+* Multi-objective optimization: [Model-Based Multi-objective Optimization: Taxonomy, Multi-Point Proposal, Toolbox and Benchmark](https://dx.doi.org/10.1007/978-3-319-15934-8_5)
+
```

Comment:
we could add this here 

First Investigations on Noisy Model-Based Multi-objective Optimization
D Horn, M Dagge, X Sun, B Bischl
International Conference on Evolutionary Multi-Criterion Optimization, 298-313


and also this

http://ieeexplore.ieee.org/document/7850221/?reload=true